### PR TITLE
Learning Rate Scheduler docstring fix

### PR DESCRIPTION
**Pull request by spzala**
Fix docstrings in Learning Rate Scheduler.

The fix can be verified by running pydocstyle path-to-file --count

Related #112593

**BEFORE the PR:**
pydocstyle torch/optim/lr_scheduler.py --count 
92 

**AFTER the PR:**
pydocstyle torch/optim/lr_scheduler.py --count 
0

cc @svekars @brycebortree

### CODE REVIEW

**Comment by spzala** [comment]
@janeyx99 @svekars I have addressed your comments to learning rate scheduler from https://github.com/pytorch/pytorch/pull/128248 I have created this separate PR for easier review per discussion there. Thanks!

File: torch/optim/lr_scheduler.py [hunk 1]
```diff
@@ -85,9 +86,13 @@ def _copy(_param):
 
 
 class LRScheduler:
+    r"""Initialize and adjust the learning rate based epoch numbers."""
```
Comment (janeyx99) [inline torch/optim/lr_scheduler.py:89]:
```suggestion
    r"""Adjusts the learning rate during optimization"""
```

**Review by janeyx99 — APPROVED** [review]

**Comment by janeyx99** [comment]
@svekars we should enable a lint if we want pydocstyle to be the way to verify docs are well written.

File: torch/optim/lr_scheduler.py (hunk 1 above)
Comment (spzala) [reply torch/optim/lr_scheduler.py:89]: Thanks so much for the quick review @janeyx99 I am updating the PR with your suggestion.
